client/my-sites/site-settings/settings-newsletter/newsletter-categories-section/newsletter-categories-section.tsx

```diff
@@ -44,6 +45,11 @@ const NewsletterCategoriesSection = ( {
 				) }
 				aria-hidden={ ! newsletterCategoriesEnabled }
 			>
+				<FormLegend>
+					{ translate(
+						'Which categories will you use for newsletter subscribers? Select all that apply:'
```

Comment:
```suggestion
						'Which categories would you like to allow people to subscribe to? Select all that apply:'
```

Just a nitpick, I think this indicates better what they are selecting. Just an opinion though 😄

Reply:
Thanks, it's a good edit. Translation already started 😅 but it could be a future improvement.